test/unittests/mongoBackend/mongoSubscribeContext_test.cpp

```diff
@@ -2887,7 +2838,7 @@ TEST(mongoSubscribeContext, matchEnt1_AttrN_CN_disjoint)
     attrsFilter.push_back("A2");
 
     EXPECT_CALL(*notifierMock, sendNotifyContextRequest(MatchNcr(&expectedNcr),"http://notify.me", "", "", "no correlator", NGSI_V1_LEGACY, attrsFilter))
-            .Times(2);
+            .Times(1);
```

Comment:
As long as we haven't lost any notifications, all should be good.
Perhaps this is a bugfix ...